src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1124,6 +1124,35 @@
     text: az feature unregister --namespace Microsoft.Compute --name GalleryPreview
 """
 
+helps['feature registration'] = """
+type: group
+short-summary: Manage resource provider feature registrations.
+"""
+
+helps['feature registration list'] = """
+type: command
+short-summary: List feature registrations.
+examples:
+  - name: List feature registrations
+    text: az feature registration list
+"""
+
+helps['feature registration create'] = """
+type: command
+short-summary: create a feature registration.
+examples:
+  - name: create the "Shared Image Gallery" feature
```

Comment:
```suggestion
short-summary: Create a feature registration.
examples:
  - name: Create the "Shared Image Gallery" feature
```

Reply:
sorry, what is your suggestion here? I did not see any difference

Reply:
The first letter needs to be capitalized

Reply:
Thanks, addressed